framerail/src/lib/server/load/logout.ts

```diff
@@ -0,0 +1,39 @@
+import defaults from "$lib/defaults"
+import { translate } from "$lib/server/deepwell/translate"
+import type { TranslateKeys } from "$lib/types"
+import { parse } from "accept-language-parser"
+
+export async function loadLogoutPage(request, cookies) {
+  // Set up parameters
+  const url = new URL(request.url)
+  const domain = url.hostname
+  const sessionToken = cookies.get("wikijump_token")
```

Comment:
Maybe worth having a `$lib/constants` for constant strings like `wikijump_token`.